invitations/admin.py

```diff
@@ -14,7 +14,7 @@
 @admin.register(Invitation)
 class InvitationAdmin(admin.ModelAdmin):
     list_display = ("email", "sent", "accepted")
-    raw_id_fields = ("inviter",)
+    autocomplete_fields = ["question"]
```

Comment:
What is the `question` field? Shouldn't this refer to `inviter` instead?

Reply:
yes, very good question 

fixed now